### PR TITLE
Basic Implement of Blade Template

### PR DESCRIPTION
**Pull request by darron1217**
### What does this implement/fix? Explain your changes.

Add basic Laravel Blade template support.
All the HTML tags in the .blade.php files are now able to be beautified.

### Does this close any currently open issues?

Issue #448
and pull request #1817

### Checklist

Check all those that are applicable and complete.

- [x] Merged with latest `master` branch
- [x] Regenerate documentation with `npm run docs`
- [x] Add change details to `CHANGELOG.md` under "Next" section
- [x] Added examples for testing to [examples/ directory](examples/)
- [x] Travis CI passes (Mac support)
- [x] AppVeyor passes (Windows support)

### CODE REVIEW

**Comment by darron1217** [comment]
@Glavin001 Can you help me with fixing the `php-cs-fixer not found` error?
I have no idea, why it gets error....

**Comment by szeck87** [comment]
@darron1217 I'm looking into the errors with AppVeyor and Travis CI.  In the meantime, please run `npm run docs` and add what you changed to the changelog.  Thanks!

**Comment by szeck87** [comment]
@darron1217 in addition to the above, please also add `"blade"` to allLanguages here:  https://github.com/darron1217/atom-beautify/blob/blade/spec/beautify-languages-spec.coffee#L47

**Comment by szeck87** [comment]
@darron1217 my bad, please also add `language-blade` to this file:  https://github.com/darron1217/atom-beautify/blob/blade/atom-packages.txt

File: examples/simple-jsbeautifyrc/blade/expected/test.blade.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  @continue
+  @endif
+  <li>{{ $user->name }}</li>
+  @if ($user->number == 5)
+  @break
+  @endif
+  @endforeach
+</body>
+
+</html>
```
Comment (szeck87) [inline examples/simple-jsbeautifyrc/blade/expected/test.blade.php]: There is a newline here on line 33, although GitHub doesn't show it very well.  Please remove it and AppVeyor should pass.  Travis will still fail due to an unrelated issue.

**Comment by szeck87** [comment]
@darron1217 please update `.travis.yml` and remove these two lines:

pip install --upgrade pip &&:  https://github.com/darron1217/atom-beautify/blob/blade/.travis.yml#L70 and
 pip2 install --upgrade pip &&:  https://github.com/darron1217/atom-beautify/blob/blade/.travis.yml#L75

That coupled with the comment under `expected/test.blade.php` should make the build pass.

File: examples/simple-jsbeautifyrc/blade/expected/test.blade.php (hunk 1 above)
Comment (darron1217) [reply examples/simple-jsbeautifyrc/blade/expected/test.blade.php]: Hmm... That's strange...
Other language tests seems also having newlines...

**Comment by darron1217** [comment]
@szeck87  Then, you mean all I need to fix is just removing `pip install --upgrade pip &&` and `pip2 install --upgrade pip` from `.travis.yml`?

File: examples/simple-jsbeautifyrc/blade/expected/test.blade.php (hunk 1 above)
Comment (darron1217) [reply examples/simple-jsbeautifyrc/blade/expected/test.blade.php]: Sorry, not for html... haha
I'll fix it :)

Comment (szeck87) [reply examples/simple-jsbeautifyrc/blade/expected/test.blade.php]: It was a specific issue with these two example files.  The original has a newline, but when beautified that line space was taken up.

**Comment by darron1217** [comment]
@Glavin001 Hello, I'm waiting for this PR to be accepted
You may be busy, but please take look at this PR if you have a minute :)

**Comment by Glavin001** [comment]
Published to v0.32.3 🎉

**Pull request merged**